### PR TITLE
79 create screenshot

### PR DESCRIPTION
**Pull request by Cozloschi**
@vtemian

### CODE REVIEW

**Comment by vtemian** [comment]
Solve conflicts first.

**Comment by vtemian** [comment]
Remove static files

File: provisioning/provision.yml [hunk 1]
```diff
@@ -51,9 +51,15 @@
     - name: Install Bower
       npm: name=bower global=yes
 
+    - name: Install phantomJs [ screenshot ]
+      npm: name=phantomjs global=yes
+
     - name: install gunicorn
       pip: name=gunicorn virtualenv={{ project_home }}
 
+    - name: Install Selenium [ screenshot ]
```
Comment (vtemian) [inline provisioning/provision.yml]: Why do you need `Selenium`?

Comment (Cozloschi) [inline provisioning/provision.yml]: I use it to import webdriver for PhantomJs.

**Comment by Cozloschi** [comment]
@vtemian

File: src/dashboard/views.py [hunk 2]
```diff
@@ -48,10 +49,18 @@ class ProviderView(View):
 
     def get(self, request, provider):
         provider_obj = get_object_or_404(Provider, name=provider)
+<<<<<<< HEAD
```
Comment (vtemian) [inline src/dashboard/views.py]: Solve those conflicts

File: src/dashboard/views.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 from src.dashboard.models import Provider, Website
 from src.dashboard.utils import Compute
```
Comment (vtemian) [inline src/dashboard/views.py]: Check imports
```

File: src/dashboard/views.py [hunk 4]
```diff
@@ -48,10 +49,18 @@ class ProviderView(View):
 
     def get(self, request, provider):
         provider_obj = get_object_or_404(Provider, name=provider)
+<<<<<<< HEAD
+        websites_obj = Website.objects.filter(provider=provider_obj.id)
+        websites_list = []
+        for website in websites_obj:
+            websites_list.append({'name': website.name, 'url': website.url, 'image': get_tld(website.url) + '.png'})
```
Comment (vtemian) [inline src/dashboard/views.py]: use list comprehension

File: src/dashboard/models.py [hunk 5]
```diff
@@ -41,7 +44,15 @@ def __str__(self):
         return self.name
 
     def save(self, *args, **kwargs):
+        phantom = webdriver.PhantomJS()
+        phantom.get(self.url)
+        phantom.set_window_size(settings.SCREENSHOT_WIDTH, settings.SCREENSHOT_HEIGHT)
+        phantom.save_screenshot(settings.BASE_DIR + '/static/screenshots/' + get_tld(self.url) + '.png')
```
Comment (vtemian) [inline src/dashboard/models.py]: Move screenshot path in settings and use string formating to build the path

**Comment by vtemian** [comment]
LGTM

**Pull request merged**